src/code/views/simulation-panel-view.coffee

```diff
@@ -92,11 +100,17 @@ module.exports = React.createClass
             (input {
               className: 'left'
               type: "number"
-              size: "3"
-              value: 1
-              min: "0"}
-            )
-            (Dropdown {anchor: "Year", items: []})
+              style: {width: "#{Math.max 3, (@state.stepSize.toString().length+1)}em"}
+              value: @state.stepSize
+              min: "0"
+              onChange: @setStepSize
+            })
+            (Dropdown {
+              isActionMenu: false
+              onSelect: SimulationStore.actions.setStepUnits
```

Comment:
The diff makes it look like there are two DropDowns defined there, but the `onChange` actually belongs to the input defined above, while `onSelect` is for the DropDown.

Do you think DropDown should call `onChange` instead of `onSelect`? I've been going back and forth whether form-like components should have a call that's semantically meaningful to the type of action, or whether they should all use a generic `onChange`. The ValueSlider, for instance, now accepts both `onValueChange` and `onRangeChange`, but they could both be melded into `onChange`.